pylint/constants.py

```diff
@@ -54,6 +56,51 @@
 DEFAULT_PYLINT_HOME = platformdirs.user_cache_dir("pylint")
 
 
+def get_pylint_home() -> str:
```

Comment:
Might want to make this private?
```suggestion
def _get_pylint_home() -> str:
```

Reply:
https://github.com/PyCQA/pylint/pull/6468/commits/4e2aac265e4fa0c7d2d95aebcec8c084de5ca4d3